modular_zubbers/code/modules/clothing/under/misc.dm

```diff
@@ -94,3 +98,646 @@
+	desc = "The staple of any bunny themed janitors. The stockings are made of cotton to allow for easy laundering."
+	icon_state = "bunnysuit_janitor"
+	inhand_icon_state = null
+	icon = 'modular_zubbers/icons/obj/clothing/under/costume.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/costume.dmi'
+	body_parts_covered = CHEST|GROIN|LEGS
+	alt_covers_chest = TRUE
+
+/obj/item/clothing/under/rank/civilian/janitor/bunnysuit/Initialize(mapload)
+	. = ..()
+
+	create_storage(storage_type = /datum/storage/pockets/tiny)
+
+/obj/item/clothing/under/rank/civilian/bartender_bunnysuit
+	name = "bartender's bunnysuit"
+	desc = "The staple of any bunny themed bartenders. Looks even more stylish than the standard bunny suit."
+	icon_state = "bunnysuit_bar"
+	inhand_icon_state = null
+	icon = 'modular_zubbers/icons/obj/clothing/under/costume.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/costume.dmi'
+	body_parts_covered = CHEST|GROIN|LEGS
+	alt_covers_chest = TRUE
+	custom_price = PAYCHECK_CREW
+
+/obj/item/clothing/under/rank/civilian/bartender_bunnysuit/Initialize(mapload)
+	. = ..()
+
+	create_storage(storage_type = /datum/storage/pockets/tiny)
+
+/obj/item/clothing/under/rank/civilian/cook_bunnysuit
+	name = "cook's bunny suit"
+	desc = "The staple of any bunny themed chefs. Shame there aren't any fishnets."
+	icon_state = "bunnysuit_chef"
+	inhand_icon_state = null
+	icon = 'modular_zubbers/icons/obj/clothing/under/costume.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/costume.dmi'
+	body_parts_covered = CHEST|GROIN|LEGS
+	alt_covers_chest = TRUE
+
+/obj/item/clothing/under/rank/civilian/cook_bunnysuit/Initialize(mapload)
+	. = ..()
+
+	create_storage(storage_type = /datum/storage/pockets/tiny)
+
+/obj/item/clothing/under/rank/civilian/hydroponics/bunnysuit
+	name = "botanist's bunny suit"
+	desc = "The staple of any bunny themed botanists. The stockings are made of faux-denim to mimic the look of overalls."
+	icon_state = "bunnysuit_botany"
+	inhand_icon_state = null
+	icon = 'modular_zubbers/icons/obj/clothing/under/costume.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/costume.dmi'
+	body_parts_covered = CHEST|GROIN|LEGS
+	alt_covers_chest = TRUE
+
+/obj/item/clothing/under/rank/civilian/hydroponics/bunnysuit/Initialize(mapload)
+	. = ..()
+
+	create_storage(storage_type = /datum/storage/pockets/tiny)
+
+/obj/item/clothing/under/rank/civilian/clown/clown_bunnysuit
+	name = "clown's bunny suit"
+	desc = "The staple of any bunny themed clowns. Now this is just ridiculous."
+	icon_state = "bunnysuit_clown"
+	inhand_icon_state = null
+	icon = 'modular_zubbers/icons/obj/clothing/under/costume.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/costume.dmi'
+	body_parts_covered = CHEST|GROIN|LEGS
+	alt_covers_chest = TRUE
+
+/obj/item/clothing/under/rank/civilian/clown_bunnysuit/Initialize(mapload)
+	. = ..()
+
+	create_storage(storage_type = /datum/storage/pockets/tiny)
```

Comment:
I'm not sure how this works with the parent already calling create_storage, I'm guessing it works fine in this case? (Pls test)

Reply:
Whoops, forgot to change that when I made it inheret the clown stuff.